class.jetpack.php

```diff
@@ -4040,19 +4014,29 @@ function admin_page_load() {
 					$error = $registered->get_error_code();
 					Jetpack::state( 'error', $error );
 					Jetpack::state( 'error', $registered->get_error_message() );
-					$this->tracking->record_user_event( 'jpc_register_fail', array(
-						'error_code' => $error,
-						'error_message' => $registered->get_error_message()
-					) );
+
+					/**
+					 * Jetpack registration Error.
+					 *
+					 * @since 7.5.0
+					 *
+					 * @param \WP_Error $error The error object.
+					 */
+					do_action( 'jetpack_connection_register_fail', $error );
```

Comment:
This is missing the second argument, `$registered`

```suggestion
					do_action( 'jetpack_connection_register_fail', $error, $registered );
```